Packs/Automox/Playbooks/Upload_Vulnerabilities_to_Automox.yml

```diff
@@ -0,0 +1,362 @@
+id: Upload Vulnerabilities to Automox
+version: -1
+fromversion: 6.0.0
+name: Upload Vulnerabilities to Automox
+description: This sub-playbook takes the entryId of a vendor-generated vulnerability report CSV and uploads it to Automox for remediation.
```

Comment:
```suggestion
description: This sub-playbook takes the entryId of a vulnerability report CSV and uploads it to Automox for remediation.
```

Doesn't exactly have to be vendor generated. Could come from anywhere really plus avoids "what vendors do this" questions. Do we need to call out where the report/file is stored in XSOAR at all or is that clear already?

Reply:
I'm good with ^